pkg/controller/mysqlbackup/mysqlbackup_controller.go

```diff
@@ -102,6 +102,7 @@ type ReconcileMysqlBackup struct {
 // Automatically generate RBAC rules to allow the Controller to read and write Deployments
 // +kubebuilder:rbac:groups=batch,resources=jobs,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups=mysql.presslabs.org,resources=mysqlbackups;mysqlbackups/status,verbs=get;list;watch;create;update;patch;delete
+// +kubebuilder:rbac:groups=mysql.presslabs.org,resources=mysqlbackups/finalizers,verbs=update
```

Comment:
```suggestion
// +kubebuilder:rbac:groups=mysql.presslabs.org,resources=mysqlbackups;mysqlbackups/status;mysqlbackups/finalizers,verbs=get;list;watch;create;update;patch;delete
```

Reply:
It seems that we only need `Update` priviliage of finalizers.